GPS_convert_fit_to_csv.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 17 09:22:37 2022
+
+@author: Eric.Honert
+
+The purpose of this code is to convert a .fit file from a GPS watch to a .csv
+file. This can reduce processing time (minimally) and allow for custom croping
+of the .fit file.
+"""
+
+# Import libraries
+from fitparse import FitFile
+import pandas as pd
+import os
+import numpy as np
+
+# Read in files
+# only read .fit files for this work
+fPath = 'C:\\Users\eric.honert\\Boa Technology Inc\\PFL Team - General\\Testing Segments\\EndurancePerformance\\TrailRun_2022\\WatchData\\fitFiles\\'
```

Comment:
Just a note this file path now needs \\OutdoorData\\ between TrailRun_2022 and fitFiles